frontend/src/assets/styles/default-theme.scss

```diff
@@ -1,33 +1,36 @@
 @use 'sass:map';
 @use '@bcgov-nr/nr-fsa-theme/design-tokens/light-theme.scss' as lightTheme;
-@use '@bcgov-nr/nr-fsa-theme/design-tokens/light-buttons.scss' as lightButtons;
+@use '@bcgov-nr/nr-fsa-theme/design-tokens/light-buttons.scss' as lightButton;
+@use '@bcgov-nr/nr-fsa-theme/design-tokens//dark-theme.scss' as darkTheme;
+@use '@bcgov-nr/nr-fsa-theme/style-sheets/primevue-components-overrides.scss';
+
 
 // Header
 $header-height: 3rem;
 $header-width: 100vw;
 
-// Main Navigation
-$text-error: map.get(lightTheme.$light-theme, 'text-error');
-$text-primary: map.get(lightTheme.$light-theme, 'text-primary');
-$text-secondary: map.get(lightTheme.$light-theme, 'text-secondary');
-
-// Buttons
-$light-button-tertiary: map.get(
-    lightButtons.$light-button-token-overrides,
-    'button-tertiary'
-);
-
-$light-button-danger-primary: map.get(
-    lightButtons.$light-button-token-overrides,
-    'button-danger-primary'
-);
-
-$light-button-danger-hover: map.get(
-    lightButtons.$light-button-token-overrides,
-    'button-danger-hover'
-);
-
-// icon
-$light-theme-icon: map.get(lightTheme.$light-theme, 'background-brand');
-$light-theme-error-icon: map.get(lightTheme.$light-theme, 'support-error');
-$light-theme-sidenav-icon: map.get(lightTheme.$light-theme, 'interactive');
\ No newline at end of file
+//darkTheme
+$dark-text-primary: map.get(darkTheme.$dark-theme, 'text-primary');
+
+//lightTheme
+$light-layer-selected-01: map.get(lightTheme.$light-theme, 'layer-selected-01');
+
+//border
+$light-border-interactive: map.get(lightTheme.$light-theme, 'border-interactive');
+$border-subtle-00: map.get(lightTheme.$light-theme, 'border-subtle-00');
+
+//text
+$light-text-primary: map.get(lightTheme.$light-theme, 'text-primary');
+$light-text-secondary: map.get(lightTheme.$light-theme, 'text-secondary');
+$light-text-error: map.get(lightTheme.$light-theme, 'text-error');
+
+//button
+$light-button-tertiary: map.get(lightButton.$light-button-token-overrides, 'button-tertiary');
+$light-button-danger-primary: map.get(lightButton.$light-button-token-overrides,'button-danger-primary');
+$light-button-danger-hover: map.get(lightButton.$light-button-token-overrides,'button-danger-hover');

```

Comment:
Hi Jota, let's check these variables if we actually use them anywhere? If not then we don't need to define them.

Reply:
hello Catherine, I just removed the unused ones. Thanks!